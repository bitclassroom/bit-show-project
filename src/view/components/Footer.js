import React from 'react'

const Footer = () => (
    <footer className="page-footer blue darken-2">
        <div className="footer-copyright">
            <div className="container">Â© {new Date().getFullYear()} Copyright hyt</div>
        </div>
    </footer>
)

export default Footer
