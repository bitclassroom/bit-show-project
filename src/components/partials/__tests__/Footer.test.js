import React from 'react'
import renderer from 'react-test-renderer'
import Footer from '../Footer'

describe('Footer', () => {
    it('should match the snapshot', () => {
        const componentTree = renderer.create(<Footer />).toJSON()
        expect(componentTree).toMatchSnapshot()
    })
})
