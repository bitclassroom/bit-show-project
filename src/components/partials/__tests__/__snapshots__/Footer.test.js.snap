// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match the snapshot 1`] = `
<footer
  className="page-footer blue darken-2"
>
  <div
    className="footer-copyright"
  >
    <div
      className="container"
    >
      Â© 
      2018
       Copyright BIT
    </div>
  </div>
</footer>
`;
